actions/index.js

```diff
@@ -21,4 +21,9 @@ export const clearFlyEntry = () => ({
 export const setCurrentFly = (data) => ({
 	type: 'SET_CURRENT_FLY',
 	data
+})
+
+export const setSelectedFlyId = (data) => ({
+	type: 'SET_FLY_ID',
+	data
```

Comment:
This is a good way to store the current Fly Id in the Redux store.  This will make the Fly Id accessible in any component in the application.  So, no need for passing props around when navigating to a different components.